crates/server/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "graph_server"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+arrow = "8.0.0"
```

Comment:
Is the rust crate nicer than the java lib? :see_no_evil:

Reply:
hehe .. not sure .. but the nice thing with the :crab: impl is the zero-overhead for the arrow arrays, they are basically wrappers around chunks of a vec. So there is no extra allocation when going from the algo result to arrow. That's different in GDS :/

Reply:
That's cool to know! IIRC did DV say that he was losing perf on the conversion between the two? I think I had wondered if we could utilise their data structure instead of ours to minimise the loss, but sounds like this is a freebie given the implementation in Rust?

Reply:
Maybe it's also costly in Rust if we wanna wrap more complex types per node e.g. vectors. But for Java converting is mandatory. I think that's also why we thought about representing (node) properties in GDS directly in Arrow data structures.